frontend/src/pages/landing/Landing.tsx

```diff
@@ -1,10 +1,70 @@
-import { redirect } from "react-router-dom";
+import { KanbanTemplate } from "@maykin-ui/admin-ui";
+import { useLoaderData } from "react-router-dom";
 
+import { KanbanCard } from "../../components/KanbanCard/KanbanCard";
+import {
+  DestructionList,
+  listDestructionLists,
+} from "../../lib/api/destructionLists";
+import { deslugify, timeAgo } from "../../lib/string";
 import "./Landing.css";
 
-/**
- * React Router loader.
- * @param request
- * TOOD: Requires destruction list lists endpoint.
- */
-export const landingLoader = () => redirect("/destruction-lists/create");
+interface LandingLoaderReturn {
+  [key: string]: DestructionList[];
+}
+
+export const landingLoader = async (): Promise<LandingLoaderReturn> => {
+  const lists = await listDestructionLists();
+  const statusMap: LandingLoaderReturn = {};
+
+  lists.forEach((list) => {
+    if (!statusMap[list.status]) {
+      statusMap[list.status] = [];
+    }
+    statusMap[list.status].push(list);
+  });
```

Comment:
Currently, if all the cards have the same status, only one column is displayed. The empty columns are not shown.

In added another comment about the statuses, I think with that you can also address this problem.